### PR TITLE
checker for column reduced cost calculation

### PR DESCRIPTION
**Pull request by enjoyUrFruits**

### CODE REVIEW

File: test/.222-revise-exit-code [hunk 1]
```diff
@@ -0,0 +1 @@
+222
```
Comment (vitornesello) [inline test/.222-revise-exit-code:1]: file committed by mistake

File: src/Algorithm/colgen/printer.jl [hunk 2]
```diff
@@ -63,9 +63,20 @@ ColGen.compute_sp_init_db(ctx::ColGenPrinterContext, sp::Formulation{DwSp}) = Co
 ColGen.set_of_columns(ctx::ColGenPrinterContext) = ColGen.set_of_columns(ctx.inner)
 
 function _calculate_column_reduced_cost(reform, col_ids)
```
Comment (guimarqu) [inline src/Algorithm/colgen/printer.jl:65]: Can you rename this function `_print_column_reduced_costs` and create a function `_calculate_column_reduced_cost(reform, col_id)` that calculates the reduced of one column only?

File: src/Algorithm/colgen/printer.jl [hunk 3]
```diff
@@ -63,9 +63,20 @@ ColGen.compute_sp_init_db(ctx::ColGenPrinterContext, sp::Formulation{DwSp}) = Co
...
-    @show typeof(reform)
+    master = reform.master
```
Comment (guimarqu) [inline src/Algorithm/colgen/printer.jl:66]:
```suggestion
    master = getmaster(reform)
```

File: src/Algorithm/colgen/printer.jl [hunk 4]
```diff
@@ -63,9 +63,20 @@ ColGen.compute_sp_init_db(ctx::ColGenPrinterContext, sp::Formulation{DwSp}) = Co
...
     @show col_ids
-    error("Good luck")
+    matrix = getcoefmatrix(master)
+    for col_id in col_ids
+        c = getcurcost(master, col_id)
+        tmp = 0
+        for (constrid, coef) in @view matrix[:, col_id] #retrieve the original cost
+            tmp += coef*getcurincval(master, constrid)
```
Comment (guimarqu) [inline src/Algorithm/colgen/printer.jl:73]:
```suggestion
            tmp += coef * getcurincval(master, constrid)
```

**Review by guimarqu — APPROVED** [review]

**Pull request merged**